internal/fs/dir_handle.go

```diff
@@ -48,23 +66,23 @@ type dirHandle struct {
 	// INVARIANT: For each i, entries[i+1].Offset == entries[i].Offset + 1
 	//
 	// GUARDED_BY(Mu)
-	entries []fuseutil.Dirent
+	Entries []fuseutil.Dirent
 
 	// Has entries yet been populated?
 	//
 	// INVARIANT: If !entriesValid, then len(entries) == 0
 	//
 	// GUARDED_BY(Mu)
-	entriesValid bool
+	EntriesValid bool
```

Comment:
why are all these properties made capital.

Reply:
For unit testing ,I had to export these from fs package

Reply:
if the test file is in same package, why do you need to make them public?

Reply:
test file is in the fs_test package just like the other test files

Reply:
Reverted it .Using setter and getter methods now.